dkist/tests/test_benchmarks.py

```diff
@@ -35,3 +32,9 @@ def plot_and_save_fig(ds=visp_dataset_no_headers, axes=axes):
         ds.plot(plot_axes=axes)
         plt.savefig("tmpplot")
         plt.close()
+
+
+@pytest.mark.benchmark
+def test_dataset_compute_data(benchmark, real_visp):
```

Comment:
As an additional benchmark it would be interesting to try one which doesn't load the whole file I.e. slice in either wavelength or slit axes.